apps/framework-cli/src/cli.rs

```diff
@@ -152,33 +152,48 @@ async fn top_command_handler(
                     if !is_git_repo {
                         crate::utilities::git::create_init_commit(project_arc, dir_path);
                         show_message!(
-                            MessageType::Success,
+                            MessageType::Info,
                             Message {
                                 action: "Init".to_string(),
                                 details: "Created a new git repository".to_string(),
                             }
                         );
                     }
 
-                    show_message!(
-                        MessageType::Success,
-                        Message {
-                            action: "Created".to_string(),
-                            details: "new project".to_string(),
-                        }
-                    );
+                    {
```

Comment:
Why did you put it in a closure?

It doesn't feel like that should be necessary

Reply:
@tg339 can better explain but it was basically the fastest workaround for avoiding duplicate TERM imports with the back to back !show_message macros